Makefile

```diff
@@ -184,7 +184,11 @@ test: vendor $(ADDON_ASSET_FILE)
 .PHONY: integration
 integration: $(MINISHIFT_BINARY)
 	mkdir -p $(INTEGRATION_TEST_DIR)
-	go test -timeout $(TIMEOUT) $(REPOPATH)/test/integration --tags=integration -v -args --test-dir $(INTEGRATION_TEST_DIR) --binary $(MINISHIFT_BINARY) $(GODOG_OPTS)
+	@if [ "$(GODOG_OPTS)" != "-tags basic" ]; then \
```

Comment:
@amitkrout , @agajdosi , could any one of you verify this change? I am not able to run test locally but tested this logic.